normative/dtpr-stickers
# Adds stickers exporting feature

# Review App
- Access new feature at Heroku Review APP
  - [Sensor Print](https://dtpr-sticker-chore-sens-htxfmu.herokuapp.com/lggYZyUxEV2u9EGHtZ83F/print)
  - [Sensor](https://dtpr-sticker-chore-sens-htxfmu.herokuapp.com/lggYZyUxEV2u9EGHtZ83F)

# Notion card link(s)
- [SVG export / copy](https://www.notion.so/normative/SVG-export-copy-6f3aea22d4df4054ba8296d776b6109c)

# PR Description (Proposed changes, approach, gotchas, etc)
- [x]  Implement new designs
- [x]  Sensor Print View refactor
- [x]  Export zip file containing:
    - [x] Full badges as PNGs
      - [x] Company logo
      - [x] Sensor
      - [x] QR Code
    - [x] Singles files
      - [x] QR code
      - [x] Hexagons
      - [x] Sensor Icon
    - [x] Mixed files
      - [x] QR Code + Text
      - [x] Sensor Icon + Text 
- [x] Refactor `IconBadge` component

# How to test it
- Run into your machine, navigate to `/sensors/<sensorId>/print`
- Click to download button then verify the zip file exported

# EXCEPTION
- Exports fully working only on Chrome or similar engine browsers because of library used
